moesif/plugins/log.lua

```diff
@@ -360,11 +382,19 @@ function _M.log_response(handler)
                             5000
                         )
 
-                        -- Save app config response
-                        app_config = app_config_body
-
-                        if debug then 
-                            handler:logDebug("[moesif] successfully fetched the application configuration")
+                        if app_config_body ~= nil and type(app_config_body) == "string" and string.sub(app_config_body, 1, 1) == "{"  then 
```

Comment:
you can use is_valid_json fcn here?

Reply:
We could but in some case if the response received is an array, we want to fail in case of application config as it should never be an array. So avoided using is_valid_json function here.